src/styles/stars.css

```
@@ -0,0 +1,37 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--1
+.stars__star:nth-child(-n+1) {
```

```suggestion
.stars__star:nth-child(-n+1),
.stars--2
.stars__star:nth-child(-n+2),
... {
```
Consider using commas to group your selectors